src/serve.rs

```diff
@@ -19,6 +19,8 @@ use crate::common::SERVER;
 use crate::config::RtcServe;
 use crate::proxy::{ProxyHandlerHttp, ProxyHandlerWebSocket};
 use crate::watch::WatchSystem;
+use axum::http::header::HeaderName;
+use std::collections::HashMap;
```

Comment:
Let's move these imports up to where their other related imports take place. `axum::http::` is already being imported, so let's group those. Then for `std::collections::HashMap`, let's move that up to line 1 (alphabetical order).